src/mbgl/annotation/annotation_tile.cpp

```diff
@@ -19,6 +19,10 @@ mapbox::util::optional<Value> AnnotationTileFeature::getValue(const std::string&
     return mapbox::util::optional<Value>();
 }
 
+std::unordered_map<std::string, std::string> AnnotationTileFeature::getAllValues() const {
```

Comment:
Shouldn't we implement the querying part in the Tile object itself rather than just stealing the features from it?